### ISSUE TITLE
Pre-work Checklist: Developer: Nikileshwari Prasadh

### ISSUE THREAD

**Issue by Thinking-Panda** [hackforla/website#3916]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by Thinking-Panda** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
5 days

**Comment by Thinking-Panda** [comment]
Progress: Currently I am working on a medium size issue to redesign a page for different viewports
Blockers: I had trouble with pull upstream and pushing with another issue earlier and that was sorted out.
Availability: M-F 10AM - 3PM
ETA: The pre-work checklist has been completed. I am looking to complete the medium issue(3983) that I am working on in a couple of days (3/13).
Pictures (optional): Before changes
![Before_changes_to_header](https://user-images.githubusercontent.com/122488603/224425179-fadf12b6-87bf-4ff8-b825-2850a1321e13.jpg)

After changes
![Link_Added](https://user-images.githubusercontent.com/122488603/224425301-863dde45-d08e-43ca-899d-f9c37f966060.jpg)

**Comment by ExperimentsInHonesty** [comment]
@Thinking-Panda I am closing this because you already got the hang of doing updates in your issues, which is the last step in the prework.  Please don't take on any more medium issues when you finish with this one. Instead, let us know you are looking for a large issue and we will find one for you.

**Comment by Thinking-Panda** [comment]
Sure. Thank you!

**Comment by HackforLABot** [comment]
## Activity Log: Thinking-Panda
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- Thinking-Panda issue closed by PR 3961: https://github.com/hackforla/website/issues/2806#event-6313998924 at 03/27/2022, 10:19 AM PDT
- Thinking-Panda issue closed by PR 3961: https://github.com/hackforla/website/issues/2806#event-7014552338 at 07/18/2022, 11:49 AM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/issues/3916 at 01/31/2023, 08:58 PM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/3916 at 01/31/2023, 09:06 PM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/3872 at 02/03/2023, 11:13 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3872#issuecomment-1419784647 at 02/06/2023, 01:30 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/pull/3954 at 02/07/2023, 03:05 PM PST
- Thinking-Panda PR was merged: https://github.com/hackforla/website/pull/3954#event-8474194073 at 02/08/2023, 04:13 PM PST
- Thinking-Panda issue closed by PR 3954: https://github.com/hackforla/website/issues/3872#event-8474194242 at 02/08/2023, 04:13 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3916#issuecomment-1424763039 at 02/09/2023, 12:23 PM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/2806#issuecomment-1308049061 at 02/10/2023, 11:12 AM PST
- Thinking-Panda opened: https://github.com/hackforla/website/pull/3961 at 02/10/2023, 12:09 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/3961#issuecomment-1430302091 at 02/14/2023, 11:59 AM PST
- Thinking-Panda PR was merged: https://github.com/hackforla/website/pull/3961#event-8523214739 at 02/15/2023, 12:11 AM PST
- Thinking-Panda issue closed by PR 3961: https://github.com/hackforla/website/issues/2806#event-8523215455 at 02/15/2023, 12:11 AM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/3059#issuecomment-1407975561 at 02/15/2023, 10:54 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3059#issuecomment-1431857827 at 02/15/2023, 10:56 AM PST
- Thinking-Panda opened: https://github.com/hackforla/website/pull/3988 at 02/16/2023, 10:59 AM PST
- Thinking-Panda PR was merged: https://github.com/hackforla/website/pull/3988#event-8543017831 at 02/16/2023, 06:48 PM PST
- Thinking-Panda issue closed by PR 3988: https://github.com/hackforla/website/issues/3059#event-8543017900 at 02/16/2023, 06:48 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/3989#issuecomment-1434040332 at 02/16/2023, 07:18 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/3989#issuecomment-1435166876 at 02/17/2023, 11:55 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/3989#pullrequestreview-1304134452 at 02/17/2023, 12:04 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4002#issuecomment-1435185572 at 02/17/2023, 12:08 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4002#pullrequestreview-1304170356 at 02/17/2023, 12:39 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4002#issuecomment-1435495608 at 02/17/2023, 10:08 PM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/3802#issuecomment-1401151372 at 02/20/2023, 11:06 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3802#issuecomment-1437441665 at 02/20/2023, 11:07 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4022#pullrequestreview-1309991535 at 02/22/2023, 11:01 AM PST
- Thinking-Panda opened: https://github.com/hackforla/website/pull/4027 at 02/22/2023, 10:10 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4027#issuecomment-1442199998 at 02/23/2023, 09:56 AM PST
- Thinking-Panda PR was merged: https://github.com/hackforla/website/pull/4027#event-8606329762 at 02/24/2023, 02:59 PM PST
- Thinking-Panda issue closed by PR 4027: https://github.com/hackforla/website/issues/3802#event-8606329940 at 02/24/2023, 02:59 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4027#issuecomment-1444838906 at 02/24/2023, 05:01 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3802#issuecomment-1445219417 at 02/25/2023, 02:34 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4048#issuecomment-1447010159 at 02/27/2023, 12:08 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4048#pullrequestreview-1316377401 at 02/27/2023, 12:26 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4056#issuecomment-1448743408 at 02/28/2023, 11:30 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4056#pullrequestreview-1320425330 at 03/01/2023, 10:43 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4056#pullrequestreview-1322750922 at 03/02/2023, 02:11 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4114#issuecomment-1456687761 at 03/06/2023, 10:12 AM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/3983#issuecomment-1445435515 at 03/06/2023, 11:49 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3983#issuecomment-1456862843 at 03/06/2023, 11:50 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4114#pullrequestreview-1327344577 at 03/06/2023, 02:57 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3916#issuecomment-1464450518 at 03/10/2023, 12:48 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3983#issuecomment-1469251406 at 03/14/2023, 08:20 PM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/pull/4197 at 03/16/2023, 01:31 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4194#issuecomment-1472896991 at 03/16/2023, 04:34 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4194#pullrequestreview-1346666389 at 03/17/2023, 01:16 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4202#issuecomment-1474419691 at 03/17/2023, 02:29 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1476708992 at 03/20/2023, 11:08 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4215#issuecomment-1476723395 at 03/20/2023, 11:17 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4215#pullrequestreview-1349173234 at 03/20/2023, 11:23 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1478303742 at 03/21/2023, 10:23 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4215#issuecomment-1478311681 at 03/21/2023, 10:28 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1481750332 at 03/23/2023, 12:08 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3983#issuecomment-1483072909 at 03/24/2023, 09:20 AM PDT
- Thinking-Panda issue closed as completed: https://github.com/hackforla/website/issues/3916#event-8848562733 at 03/26/2023, 05:49 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/3916#issuecomment-1485725534 at 03/27/2023, 12:11 PM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/issues/4490 at 04/11/2023, 08:20 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1507345270 at 04/13/2023, 10:26 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1511954603 at 04/17/2023, 12:19 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1513903110 at 04/18/2023, 04:19 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4548#issuecomment-1515343976 at 04/19/2023, 01:34 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4197#issuecomment-1515780453 at 04/19/2023, 11:28 PM PDT
- Thinking-Panda PR was merged: https://github.com/hackforla/website/pull/4197#event-9053748124 at 04/20/2023, 12:00 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4548#pullrequestreview-1394502844 at 04/20/2023, 10:53 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4560#issuecomment-1520591274 at 04/24/2023, 10:53 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4560#pullrequestreview-1398560607 at 04/24/2023, 11:11 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4560#issuecomment-1522144705 at 04/25/2023, 10:15 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4560#pullrequestreview-1400395557 at 04/25/2023, 10:25 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4561#issuecomment-1522159052 at 04/25/2023, 10:28 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4561#pullrequestreview-1400422708 at 04/25/2023, 10:45 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4569#issuecomment-1526056122 at 04/27/2023, 10:15 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4570#issuecomment-1526057050 at 04/27/2023, 10:16 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1526058278 at 04/27/2023, 10:17 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4570#issuecomment-1526428505 at 04/27/2023, 01:48 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4570#pullrequestreview-1404859728 at 04/27/2023, 01:50 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4569#pullrequestreview-1404914418 at 04/27/2023, 02:31 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4573#pullrequestreview-1406359822 at 04/28/2023, 10:44 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1527990621 at 04/28/2023, 12:19 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4573#pullrequestreview-1406550412 at 04/28/2023, 01:22 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1528661775 at 04/28/2023, 10:32 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4573#pullrequestreview-1407051296 at 04/29/2023, 09:24 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4569#issuecomment-1528937189 at 04/29/2023, 09:37 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4640#issuecomment-1541201005 at 05/09/2023, 07:30 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4640#pullrequestreview-1421206206 at 05/10/2023, 11:47 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4640#pullrequestreview-1421221955 at 05/10/2023, 11:59 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4657#issuecomment-1546019184 at 05/12/2023, 09:47 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4650#issuecomment-1546021542 at 05/12/2023, 09:49 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4657#pullrequestreview-1425015689 at 05/12/2023, 11:56 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/4439#issuecomment-1546226131 at 05/12/2023, 01:03 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4650#pullrequestreview-1427298072 at 05/15/2023, 01:11 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4680#issuecomment-1550077223 at 05/16/2023, 10:24 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4677#issuecomment-1550079956 at 05/16/2023, 10:26 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4680#pullrequestreview-1429380880 at 05/16/2023, 01:47 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4677#pullrequestreview-1429470979 at 05/16/2023, 02:34 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4650#issuecomment-1550618092 at 05/16/2023, 08:05 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4650#issuecomment-1553844472 at 05/18/2023, 05:22 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4650#pullrequestreview-1434838058 at 05/19/2023, 10:38 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4650#pullrequestreview-1435203834 at 05/19/2023, 02:49 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4650#issuecomment-1555292134 at 05/19/2023, 02:50 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4650#issuecomment-1557544068 at 05/22/2023, 09:33 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4680#pullrequestreview-1440863779 at 05/23/2023, 08:00 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4757#issuecomment-1569446150 at 05/30/2023, 08:34 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4757#pullrequestreview-1453849213 at 05/31/2023, 10:59 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4842#issuecomment-1595422494 at 06/16/2023, 04:01 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/4842#pullrequestreview-1489177883 at 06/20/2023, 03:05 PM PDT
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/4136#issuecomment-1598129971 at 06/20/2023, 06:02 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/4136#issuecomment-1599883128 at 06/20/2023, 06:04 PM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/pull/4942 at 07/12/2023, 09:53 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4942#issuecomment-1633086551 at 07/12/2023, 12:22 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4942#issuecomment-1659486362 at 07/31/2023, 07:53 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4942#issuecomment-1669356354 at 08/08/2023, 03:31 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4942#issuecomment-1674859606 at 08/11/2023, 07:11 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/4942#issuecomment-1685285211 at 08/20/2023, 06:24 AM PDT
- Thinking-Panda PR was merged: https://github.com/hackforla/website/pull/4942#event-10140193918 at 08/20/2023, 08:22 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5339#issuecomment-1698206515 at 08/29/2023, 03:03 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5339#pullrequestreview-1603710640 at 08/30/2023, 06:37 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5444#issuecomment-1707480204 at 09/05/2023, 05:33 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5444#pullrequestreview-1615952135 at 09/07/2023, 10:39 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5444#issuecomment-1710547930 at 09/07/2023, 10:42 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5444#issuecomment-1714343169 at 09/11/2023, 10:59 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5538#issuecomment-1724051798 at 09/18/2023, 10:28 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5538#pullrequestreview-1631658987 at 09/18/2023, 10:45 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5537#issuecomment-1724085148 at 09/18/2023, 10:47 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5537#pullrequestreview-1631733212 at 09/18/2023, 11:31 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5538#pullrequestreview-1634197530 at 09/19/2023, 02:13 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5557#issuecomment-1726541466 at 09/19/2023, 02:42 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5557#pullrequestreview-1634252334 at 09/19/2023, 02:48 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5658#issuecomment-1746010710 at 10/03/2023, 06:58 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5658#pullrequestreview-1660248052 at 10/05/2023, 09:09 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5692#issuecomment-1756619549 at 10/10/2023, 07:04 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5690#issuecomment-1756620604 at 10/10/2023, 07:06 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5692#pullrequestreview-1674966304 at 10/12/2023, 12:08 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5690#pullrequestreview-1675072283 at 10/12/2023, 12:55 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5692#pullrequestreview-1680760592 at 10/16/2023, 11:51 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5692#issuecomment-1765109884 at 10/16/2023, 12:04 PM PDT
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/4188 at 10/16/2023, 09:02 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/4188#issuecomment-1765628147 at 10/16/2023, 09:03 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5733#issuecomment-1769034310 at 10/18/2023, 10:41 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5733#pullrequestreview-1685819594 at 10/18/2023, 11:20 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/4188#issuecomment-1774101539 at 10/22/2023, 06:52 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/4188#issuecomment-1792904147 at 11/03/2023, 11:05 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/4188#issuecomment-1809224454 at 11/13/2023, 02:17 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5891#issuecomment-1815163866 at 11/16/2023, 11:17 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5891#pullrequestreview-1735453010 at 11/16/2023, 12:45 PM PST
- Thinking-Panda closed by PR 1: https://github.com/hackforla/website/issues/4188#event-11030492802 at 11/21/2023, 04:17 PM PST
- Thinking-Panda reopened: https://github.com/hackforla/website/issues/4188#event-11030492802 at 11/21/2023, 07:53 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/pull/5929 at 11/21/2023, 08:11 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5929#issuecomment-1823324444 at 11/22/2023, 11:01 AM PST
- Thinking-Panda opened: https://github.com/hackforla/website/pull/5932 at 11/22/2023, 11:12 AM PST
- Thinking-Panda closed: https://github.com/hackforla/website/pull/5929#event-11040286088 at 11/22/2023, 11:13 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5932#issuecomment-1823431486 at 11/22/2023, 11:58 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5941#issuecomment-1830873777 at 11/28/2023, 02:41 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5941#pullrequestreview-1754087299 at 11/28/2023, 02:46 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5940#issuecomment-1831137870 at 11/28/2023, 07:01 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/5940#pullrequestreview-1756251777 at 11/29/2023, 01:31 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5932#issuecomment-1834170088 at 11/30/2023, 09:00 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/5932#issuecomment-1839338910 at 12/04/2023, 11:32 AM PST
- Thinking-Panda PR was merged: https://github.com/hackforla/website/pull/5932#event-11186320584 at 12/07/2023, 12:02 PM PST
- Thinking-Panda issue closed by PR 1: https://github.com/hackforla/website/issues/4188#event-11186320834 at 12/07/2023, 12:02 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6098#issuecomment-1887860904 at 01/11/2024, 11:50 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6098#pullrequestreview-1820262290 at 01/14/2024, 08:25 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6190#issuecomment-1917786904 at 01/30/2024, 11:56 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6190#pullrequestreview-1852255524 at 01/30/2024, 12:09 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6148#issuecomment-1917916371 at 01/30/2024, 01:22 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6197#issuecomment-1917927971 at 01/30/2024, 01:30 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6197#pullrequestreview-1852430991 at 01/30/2024, 01:47 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6198#issuecomment-1917964297 at 01/30/2024, 01:58 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6198#pullrequestreview-1852585481 at 01/30/2024, 03:20 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6148#pullrequestreview-1852849691 at 01/30/2024, 07:55 PM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/5999 at 02/01/2024, 02:10 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/5999#issuecomment-1927998305 at 02/05/2024, 11:59 AM PST
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6234 at 02/05/2024, 12:26 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6235 at 02/05/2024, 12:32 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6236 at 02/05/2024, 12:36 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6237 at 02/05/2024, 12:39 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6238 at 02/05/2024, 12:43 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6239 at 02/05/2024, 12:46 PM PST
- Thinking-Panda issue closed as completed: https://github.com/hackforla/website/issues/5999#event-11711399192 at 02/05/2024, 04:13 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/5999#issuecomment-1930581920 at 02/06/2024, 11:06 AM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/6225 at 02/06/2024, 02:23 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6260#issuecomment-1930883460 at 02/06/2024, 02:34 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6260#pullrequestreview-1866484124 at 02/06/2024, 03:05 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6260#pullrequestreview-1868625312 at 02/07/2024, 11:35 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6225#issuecomment-1932744099 at 02/07/2024, 11:40 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6225#issuecomment-1934851091 at 02/08/2024, 12:04 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6225#issuecomment-1934932083 at 02/08/2024, 01:05 PM PST
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6272 at 02/09/2024, 12:57 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6272#issuecomment-1936593657 at 02/09/2024, 12:58 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6225#issuecomment-1939341412 at 02/12/2024, 10:52 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6287#issuecomment-1942479887 at 02/13/2024, 12:51 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6287#pullrequestreview-1878914798 at 02/13/2024, 01:02 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6281#issuecomment-1942677857 at 02/13/2024, 01:46 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6281#pullrequestreview-1879006440 at 02/13/2024, 01:58 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6287#issuecomment-1942850162 at 02/13/2024, 03:30 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6324#issuecomment-1955152565 at 02/20/2024, 01:34 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6318#pullrequestreview-1891785764 at 02/20/2024, 03:45 PM PST
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/6272 at 02/26/2024, 06:55 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6272#issuecomment-1968089171 at 02/27/2024, 06:44 PM PST
- Thinking-Panda unassigned: https://github.com/hackforla/website/issues/6272 at 02/27/2024, 07:01 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6371#issuecomment-1968107841 at 02/27/2024, 07:06 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6371#pullrequestreview-1906967565 at 02/28/2024, 10:23 AM PST
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6379#event-11955706448 at 02/28/2024, 10:31 AM PST
- Thinking-Panda closed by PR 6379: https://github.com/hackforla/website/issues/6113#event-11955706701 at 02/28/2024, 10:31 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6318#issuecomment-1969622985 at 02/28/2024, 10:49 AM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6387#issuecomment-1969627107 at 02/28/2024, 10:51 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6387#pullrequestreview-1907055626 at 02/28/2024, 11:00 AM PST
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6387#event-11956069844 at 02/28/2024, 11:00 AM PST
- Thinking-Panda closed by PR 6387: https://github.com/hackforla/website/issues/6313#event-11956070077 at 02/28/2024, 11:00 AM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6392#pullrequestreview-1915233242 at 03/04/2024, 12:00 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6414#issuecomment-1977515111 at 03/04/2024, 01:45 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6414#pullrequestreview-1915417378 at 03/04/2024, 01:47 PM PST
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6414#event-12005226525 at 03/04/2024, 01:47 PM PST
- Thinking-Panda closed by PR 6414: https://github.com/hackforla/website/issues/6258#event-12005234189 at 03/04/2024, 01:49 PM PST
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6411#issuecomment-1977525710 at 03/04/2024, 01:52 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6434#pullrequestreview-1918663464 at 03/05/2024, 07:18 PM PST
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6434#event-12021978957 at 03/05/2024, 07:18 PM PST
- Thinking-Panda closed by PR 6434: https://github.com/hackforla/website/issues/5826#event-12021979044 at 03/05/2024, 07:18 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6410#pullrequestreview-1918680570 at 03/05/2024, 07:38 PM PST
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6410#pullrequestreview-1928725445 at 03/11/2024, 11:21 AM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6410#event-12078179430 at 03/11/2024, 11:22 AM PDT
- Thinking-Panda closed by PR 6410: https://github.com/hackforla/website/issues/6376#event-12078179694 at 03/11/2024, 11:22 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6411#issuecomment-1989194208 at 03/11/2024, 11:45 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6272#issuecomment-1989200625 at 03/11/2024, 11:50 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6234#issuecomment-1989215320 at 03/11/2024, 11:58 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6392#issuecomment-1992401869 at 03/12/2024, 12:39 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6465#event-12174881975 at 03/19/2024, 12:53 PM PDT
- Thinking-Panda closed by PR 6465: https://github.com/hackforla/website/issues/6169#event-12174882189 at 03/19/2024, 12:53 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6518#pullrequestreview-1964047171 at 03/27/2024, 10:37 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6411#pullrequestreview-1964121370 at 03/27/2024, 11:02 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6225#issuecomment-2025858416 at 03/28/2024, 11:29 AM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6518#event-12285877731 at 03/28/2024, 11:32 AM PDT
- Thinking-Panda closed by PR 6518: https://github.com/hackforla/website/issues/6509#event-12285877946 at 03/28/2024, 11:32 AM PDT
- Thinking-Panda issue closed as completed: https://github.com/hackforla/website/issues/6225#event-12384886070 at 04/07/2024, 12:02 PM PDT
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/6290 at 04/18/2024, 02:01 PM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6683 at 04/18/2024, 02:19 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6683#issuecomment-2073216121 at 04/23/2024, 12:05 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6711#pullrequestreview-2018626093 at 04/23/2024, 06:46 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6690#issuecomment-2073868821 at 04/23/2024, 07:12 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6678#issuecomment-2073873384 at 04/23/2024, 07:16 PM PDT
- Thinking-Panda issue closed as completed: https://github.com/hackforla/website/issues/6290#event-12635289871 at 04/28/2024, 09:43 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6770#issuecomment-2083703473 at 04/29/2024, 02:25 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6770#event-12649345077 at 04/29/2024, 02:26 PM PDT
- Thinking-Panda closed by PR 6770: https://github.com/hackforla/website/issues/6763#event-12649345297 at 04/29/2024, 02:26 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6767#issuecomment-2083730958 at 04/29/2024, 02:41 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6746#issuecomment-2083760252 at 04/29/2024, 03:02 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6746#event-12649690025 at 04/29/2024, 03:02 PM PDT
- Thinking-Panda closed by PR 6746: https://github.com/hackforla/website/issues/6700#event-12649690255 at 04/29/2024, 03:02 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6782#issuecomment-2086512461 at 04/30/2024, 11:39 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6782#pullrequestreview-2032218655 at 04/30/2024, 11:41 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6782#issuecomment-2086529856 at 04/30/2024, 11:43 AM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6782#event-12662431854 at 04/30/2024, 11:43 AM PDT
- Thinking-Panda closed by PR 6782: https://github.com/hackforla/website/issues/6633#event-12662432033 at 04/30/2024, 11:43 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6686#issuecomment-2086691613 at 04/30/2024, 12:11 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6686#event-12662740601 at 04/30/2024, 12:11 PM PDT
- Thinking-Panda closed by PR 6686: https://github.com/hackforla/website/issues/6623#event-12662740902 at 04/30/2024, 12:11 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6747#issuecomment-2086800878 at 04/30/2024, 12:27 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6747#pullrequestreview-2034340798 at 05/01/2024, 12:53 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6777#pullrequestreview-2034411477 at 05/01/2024, 01:36 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6814#event-12734184291 at 05/07/2024, 12:11 PM PDT
- Thinking-Panda closed by PR 6814: https://github.com/hackforla/website/issues/6703#event-12734184511 at 05/07/2024, 12:11 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6810#issuecomment-2099129172 at 05/07/2024, 12:14 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6810#pullrequestreview-2043982054 at 05/07/2024, 12:21 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6810#event-12734293606 at 05/07/2024, 12:21 PM PDT
- Thinking-Panda closed by PR 6810: https://github.com/hackforla/website/issues/6762#event-12734293807 at 05/07/2024, 12:21 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6574#issuecomment-2123248757 at 05/21/2024, 11:53 AM PDT
- Thinking-Panda closed as completed: https://github.com/hackforla/website/issues/6574#event-12884210356 at 05/21/2024, 11:53 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6865#issuecomment-2123746489 at 05/21/2024, 07:16 PM PDT
- Thinking-Panda closed: https://github.com/hackforla/website/pull/6865#event-12887392950 at 05/21/2024, 07:16 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6747#issuecomment-2125832527 at 05/22/2024, 02:52 PM PDT
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/6320 at 05/28/2024, 11:04 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6320#issuecomment-2135834250 at 05/28/2024, 11:04 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6766#issuecomment-2136416764 at 05/28/2024, 07:44 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6320#issuecomment-2138184758 at 05/29/2024, 01:11 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6320#issuecomment-2138193502 at 05/29/2024, 01:17 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6891#event-12977053304 at 05/29/2024, 01:38 PM PDT
- Thinking-Panda closed by PR 6891: https://github.com/hackforla/website/issues/6734#event-12977053482 at 05/29/2024, 01:38 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6593#issuecomment-2138235210 at 05/29/2024, 01:46 PM PDT
- Thinking-Panda closed as completed: https://github.com/hackforla/website/issues/6593#event-12977127260 at 05/29/2024, 01:46 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6320#issuecomment-2142912885 at 05/31/2024, 01:08 PM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6920 at 05/31/2024, 01:19 PM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/issues/6930 at 06/03/2024, 11:06 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6920#issuecomment-2145830271 at 06/03/2024, 11:11 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6925#issuecomment-2150766987 at 06/05/2024, 12:07 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6925#event-13056210298 at 06/05/2024, 12:07 PM PDT
- Thinking-Panda closed by PR 6925: https://github.com/hackforla/website/issues/6725#event-13056210586 at 06/05/2024, 12:07 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6924#issuecomment-2150905742 at 06/05/2024, 01:25 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6924#event-13057117739 at 06/05/2024, 01:27 PM PDT
- Thinking-Panda closed by PR 6924: https://github.com/hackforla/website/issues/6864#event-13057117937 at 06/05/2024, 01:27 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6920#issuecomment-2155371380 at 06/07/2024, 12:06 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6747#pullrequestreview-2105323488 at 06/07/2024, 01:36 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6747#issuecomment-2155634403 at 06/07/2024, 03:07 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6920#issuecomment-2159254384 at 06/10/2024, 01:46 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6920#issuecomment-2161268259 at 06/11/2024, 10:24 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6983#pullrequestreview-2111128136 at 06/11/2024, 11:02 AM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6983#event-13120940795 at 06/11/2024, 11:02 AM PDT
- Thinking-Panda closed by PR 6983: https://github.com/hackforla/website/issues/6936#event-13120940981 at 06/11/2024, 11:02 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6737#issuecomment-2161354929 at 06/11/2024, 11:18 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6972#issuecomment-2161456220 at 06/11/2024, 12:22 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6972#event-13121767777 at 06/11/2024, 12:22 PM PDT
- Thinking-Panda closed by PR 6972: https://github.com/hackforla/website/issues/6655#event-13121767956 at 06/11/2024, 12:22 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6915#issuecomment-2161989966 at 06/11/2024, 07:34 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6594#issuecomment-2162040498 at 06/11/2024, 08:36 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6594#issuecomment-2162041113 at 06/11/2024, 08:36 PM PDT
- Thinking-Panda closed as completed: https://github.com/hackforla/website/issues/6594#event-13125413775 at 06/11/2024, 08:36 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6750#issuecomment-2163428113 at 06/12/2024, 09:10 AM PDT
- Thinking-Panda closed as completed: https://github.com/hackforla/website/issues/6750#event-13134349469 at 06/12/2024, 09:10 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6755#issuecomment-2163469758 at 06/12/2024, 09:34 AM PDT
- Thinking-Panda closed as completed: https://github.com/hackforla/website/issues/6755#event-13134610280 at 06/12/2024, 09:34 AM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/6985#pullrequestreview-2113630587 at 06/12/2024, 10:11 AM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/6985#event-13135052450 at 06/12/2024, 10:11 AM PDT
- Thinking-Panda closed by PR 6985: https://github.com/hackforla/website/issues/6910#event-13135052646 at 06/12/2024, 10:11 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6747#issuecomment-2179374097 at 06/19/2024, 12:46 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/7063#pullrequestreview-2139777515 at 06/25/2024, 01:21 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7063#event-13287463120 at 06/25/2024, 01:21 PM PDT
- Thinking-Panda closed by PR 7063: https://github.com/hackforla/website/issues/7060#event-13287463396 at 06/25/2024, 01:21 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/7059#pullrequestreview-2139795293 at 06/25/2024, 01:32 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7059#event-13287574187 at 06/25/2024, 01:32 PM PDT
- Thinking-Panda closed by PR 7059: https://github.com/hackforla/website/issues/7044#event-13287574422 at 06/25/2024, 01:32 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7001#issuecomment-2189949178 at 06/25/2024, 01:53 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6982#issuecomment-2193023726 at 06/26/2024, 07:43 PM PDT
- Thinking-Panda issue closed as completed: https://github.com/hackforla/website/issues/6320#event-13343090275 at 06/30/2024, 02:38 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/7052#pullrequestreview-2170240357 at 07/10/2024, 01:39 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7052#event-13462074115 at 07/10/2024, 01:41 PM PDT
- Thinking-Panda closed by PR 7052: https://github.com/hackforla/website/issues/6787#event-13462074328 at 07/10/2024, 01:41 PM PDT
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/6579 at 09/03/2024, 11:48 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6579#issuecomment-2327196158 at 09/03/2024, 11:49 AM PDT
- Thinking-Panda opened: https://github.com/hackforla/website/issues/7379 at 09/03/2024, 11:56 AM PDT
- Thinking-Panda assigned: https://github.com/hackforla/website/issues/6319 at 09/03/2024, 12:44 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6319#issuecomment-2327299384 at 09/03/2024, 12:44 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7376#event-14138080045 at 09/04/2024, 03:05 PM PDT
- Thinking-Panda closed by PR 7376: https://github.com/hackforla/website/issues/7286#event-14138080203 at 09/04/2024, 03:05 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7371#event-14138152608 at 09/04/2024, 03:10 PM PDT
- Thinking-Panda closed by PR 7371: https://github.com/hackforla/website/issues/7287#event-14138152740 at 09/04/2024, 03:10 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7370#event-14138224179 at 09/04/2024, 03:18 PM PDT
- Thinking-Panda closed by PR 7370: https://github.com/hackforla/website/issues/7179#event-14138224297 at 09/04/2024, 03:18 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7368#issuecomment-2330276656 at 09/04/2024, 03:44 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6319#issuecomment-2332399884 at 09/05/2024, 11:37 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/6319#issuecomment-2338890373 at 09/09/2024, 12:14 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7428#issuecomment-2342479503 at 09/10/2024, 07:16 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7369#issuecomment-2342485258 at 09/10/2024, 07:23 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7298#issuecomment-2342489842 at 09/10/2024, 07:27 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/6873#issuecomment-2342491813 at 09/10/2024, 07:30 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/7143#issuecomment-2344267767 at 09/11/2024, 10:23 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7427#issuecomment-2344319430 at 09/11/2024, 10:49 AM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7427#event-14220457353 at 09/11/2024, 10:50 AM PDT
- Thinking-Panda closed by PR 7427: https://github.com/hackforla/website/issues/7190#event-14220457555 at 09/11/2024, 10:50 AM PDT
- Thinking-Panda issue closed as completed: https://github.com/hackforla/website/issues/6579#event-14263550271 at 09/15/2024, 09:07 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7463#issuecomment-2359185445 at 09/18/2024, 11:52 AM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7463#event-14312282936 at 09/18/2024, 11:52 AM PDT
- Thinking-Panda closed by PR 7463: https://github.com/hackforla/website/issues/7413#event-14312283188 at 09/18/2024, 11:52 AM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7468#issuecomment-2359202588 at 09/18/2024, 12:01 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7462#issuecomment-2359223122 at 09/18/2024, 12:14 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7462#event-14312516561 at 09/18/2024, 12:14 PM PDT
- Thinking-Panda closed by PR 7462: https://github.com/hackforla/website/issues/7412#event-14312516777 at 09/18/2024, 12:14 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7447#issuecomment-2359232004 at 09/18/2024, 12:19 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7447#event-14312572188 at 09/18/2024, 12:19 PM PDT
- Thinking-Panda closed by PR 7447: https://github.com/hackforla/website/issues/7410#event-14312572358 at 09/18/2024, 12:19 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/7467#pullrequestreview-2313721968 at 09/18/2024, 01:25 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7467#issuecomment-2369836784 at 09/23/2024, 05:20 PM PDT
- Thinking-Panda submitted review: https://github.com/hackforla/website/pull/7467#pullrequestreview-2326278667 at 09/24/2024, 12:39 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/issues/7093#issuecomment-2372275423 at 09/24/2024, 01:09 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7496#event-14387938990 at 09/24/2024, 01:10 PM PDT
- Thinking-Panda closed by PR 7496: https://github.com/hackforla/website/issues/7093#event-14387939245 at 09/24/2024, 01:10 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7568#issuecomment-2400785845 at 10/08/2024, 01:43 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7568#event-14562668329 at 10/08/2024, 01:43 PM PDT
- Thinking-Panda closed by PR 7568: https://github.com/hackforla/website/issues/7566#event-14562668496 at 10/08/2024, 01:43 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7558#event-14562902050 at 10/08/2024, 02:08 PM PDT
- Thinking-Panda closed by PR 7558: https://github.com/hackforla/website/issues/7442#event-14562902237 at 10/08/2024, 02:08 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7616#issuecomment-2430156634 at 10/22/2024, 01:07 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7639#issuecomment-2445146730 at 10/29/2024, 12:27 PM PDT
- Thinking-Panda merged: https://github.com/hackforla/website/pull/7639#event-14990258358 at 10/29/2024, 12:27 PM PDT
- Thinking-Panda closed by PR 7639: https://github.com/hackforla/website/issues/7542#event-14990258566 at 10/29/2024, 12:27 PM PDT
- Thinking-Panda commented on: https://github.com/hackforla/website/pull/7637#issuecomment-2445692812 at 10/29/2024, 07:28 PM PDT
- Thinking-Panda unassigned: https://github.com/hackforla/website/issues/6319 at 03/16/2025, 03:50 PM PDT
